gulpfile.js

```diff
@@ -192,11 +191,20 @@ gulp.task("build:copy", () => {
 
     // Get the config.js file for the task to know what scanner to use
     const configJs = path.join(BUILD_TS_DIR, "common", commonPath, "config.js");
+
+    console.log("configJs", configJs);
     // eslint-disable-next-line import/no-dynamic-require
     const { scanner } = require(configJs);
 
     // Copy Scanner CLI
-    if (taskNeedsCliScanner(taskName)) {
+    const versionRequiresBundledScanners = ["sonarcloud@v1", "sonarqube@v4", "sonarqube@v5"];
```

Comment:
I think, instead of this, we could use `config.ts` in common folders to know whether scanners should be bundled or not, since we have access to all the `config.js` in here through `const { scanner } = require(configJs);` above